Libplanet.Tests/Blockchain/BlockChainTest.Append.cs

```diff
@@ -537,7 +539,7 @@ public void UnstageAfterAppendComplete()
                 blockInterval: TimeSpan.FromSeconds(10)
             ).Evaluate(privateKey, _blockChain);
             _blockChain.Append(block3);
-            Assert.Empty(_blockChain.GetStagedTransactionIds());
+            Assert.Equal(1, _blockChain.GetStagedTransactionIds().Count);
```

Comment:
It seems different logic... is it a bug before?

Reply:
This is due to changed behavior of `VolatileStagePolicy`.

Reply:
Is it ok? isn't it affected nine chronicles mainnet?

Reply:
From a miner's perspective, it should be fine as the changed implementation holds a superset of `Transaction<T>`s held by the previous implementation. `BlockChain<T>.GatherTransactionsToMine()` actually does the heavy lifting of filtering what can be mined.

As from a network's perspective, we don't have no known measured metric to compare whether this would increase or decrease the network's traffic. My gut feeling is that it would decrease the overall traffic.

I do need to make sure this plays nice with [lib9c]`s `StagePolicy` though and I'm currently working on it.

[lib9c]: https://github.com/planetarium/lib9c